x-pack/plugins/enterprise_search/public/applications/semantic_search/components/semantic_search_guide/semantic_search_guide.tsx

```diff
@@ -0,0 +1,355 @@
+{
+  "service": "amazonbedrock",
+  "service_settings": {
+    "access_key": "<aws_access_key>",
+    "secret_key": "<aws_secret_key>",
+    "region": "us-east-1",
+    "provider": "amazontitan",
+    "model": "amazon.titan-embed-text-v2:0"
+  }
+}
+`;
+
+const CREATE_INDEX_SNIPPET = `PUT /my-index
+{
+  "mappings": {
+    "properties": {
+      "text": {
+        "type": "semantic_text",
+        "inference_id": "my-inference-endpoint"
+      }
+    }
+  }
+}`;
+
+const INGEST_SNIPPET = `POST /my-index/_doc
+{ 
+  "text": "There are a few foods and food groups that will help to fight inflammation and delayed onset muscle soreness (both things that are inevitable after a long, hard workout) when you incorporate them into your postworkout eats, whether immediately after your run or at a meal later in the day" 
+}`;
+
+const QUERY_SNIPPET = `POST /my-index/_search
+{
+  "size" : 3,
+  "query" : {
+    "semantic": {
+      "field": "text", 
+      "query": "How to avoid muscle soreness while running?" 
+    }
+  }
+}`;
+
+const modelSelection: InferenceModel[] = [
+  {
+    id: 'elser',
+    modelName: 'ELSER',
+    code: SETUP_INFERENCE_ENDPOINT_ELSER,
+    link: docLinks.elser,
+    description: "Elastic's proprietary, best-in-class sparse vector model for semantic search.",
+  },
+  {
+    id: 'e5',
+    modelName: 'E5 Multilingual',
+    code: SETUP_INFERENCE_ENDPOINT_E5,
+    link: docLinks.e5Model,
+    description: 'Try an optimized third party multilingual model.',
+  },
+  {
+    code: SETUP_INFERENCE_ENDPOINT_OPENAI,
+    id: 'openai',
+    modelName: 'OpenAI',
+    link: 'https://platform.openai.com/docs/guides/embeddings',
+    description: "Connect with OpenAI's embedding models.",
+  },
+  {
+    id: 'bedrock',
+    modelName: 'Amazon Bedrock',
+    code: SETUP_INFERENCE_ENDPOINT_BEDROCK,
+    link: 'https://docs.aws.amazon.com/bedrock/latest/userguide/titan-embedding-models.html',
+    description: "Use Amazon Bedrock's embedding models.",
+  },
+];
+
+interface SelectModelPanelProps {
+  isSelectedModel: boolean;
+  model: InferenceModel;
+  setSelectedModel: (model: InferenceModel) => void;
+}
+
+interface InferenceModel {
+  code: string;
+  id: string;
+  link: string;
+  modelName: string;
+  description: string;
+}
+
+const SelectModelPanel: React.FC<SelectModelPanelProps> = ({
+  model,
+  setSelectedModel,
+  isSelectedModel,
+}) => {
+  return (
+    <EuiFlexItem>
+      <EuiCard
+        title={model.modelName}
+        className={
+          isSelectedModel ? 'chooseEmbeddingModelSelectedBorder' : 'chooseEmbeddingModelBorder'
+        }
+        description={
+          <>
+            <EuiText>
+              <p>{model.description}</p>
+            </EuiText>
+            <EuiSpacer size="s" />
+            <EuiLink
+              data-test-subj="enterpriseSearchSelectModelPanelReadMoreLink"
+              href={model.link}
+              target="_blank"
```

Comment:
Should  we have an `aria-label` with a more descriptive text than just the `Read more` here. (learning from our a11y bugs)